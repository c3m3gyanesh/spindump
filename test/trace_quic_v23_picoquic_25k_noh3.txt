A test run with the picoquic client and server, for a 5K file transfer using the pure QUIC mode (not HTTP3). Spinning fully.
