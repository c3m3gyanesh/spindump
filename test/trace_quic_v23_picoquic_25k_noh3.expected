[
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "08935227a1a34035-dba1d38741e64dfc (58532:4433)", "Ts": 1571077732022807, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "08935227a1a34035-08b0495f06426a29 (58532:4433)", "Ts": 1571077732101913, "State": "Up", "Right_rtt": 79106, "Packets1": 1, "Packets2": 0, "Bytes1": 1280, "Bytes2": 0, "Bandwidth1": 1280, "Bandwidth2": 0 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "935227a1a3403545-08b0495f06426a29 (58532:4433)", "Ts": 1571077732176362, "State": "Up", "Transition": "0-1", "Who": "responder", "Packets1": 5, "Packets2": 15, "Bytes1": 3354, "Bytes2": 17983, "Bandwidth1": 3354, "Bandwidth2": 17983 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "935227a1a3403545-08b0495f06426a29 (58532:4433)", "Ts": 1571077732176413, "State": "Up", "Transition": "1-0", "Who": "initiator", "Packets1": 5, "Packets2": 18, "Bytes1": 3354, "Bytes2": 21823, "Bandwidth1": 3354, "Bandwidth2": 21823 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "935227a1a3403545-08b0495f06426a29 (58532:4433)", "Ts": 1571077732176413, "State": "Up", "Left_rtt": 51, "Packets1": 5, "Packets2": 18, "Bytes1": 3354, "Bytes2": 21823, "Bandwidth1": 3354, "Bandwidth2": 21823 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "935227a1a3403545-08b0495f06426a29 (58532:4433)", "Ts": 1571077732254932, "State": "Up", "Transition": "1-0", "Who": "responder", "Packets1": 11, "Packets2": 24, "Bytes1": 3811, "Bytes2": 28734, "Bandwidth1": 3811, "Bandwidth2": 28734 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "935227a1a3403545-08b0495f06426a29 (58532:4433)", "Ts": 1571077732254932, "State": "Up", "Right_rtt": 78519, "Packets1": 11, "Packets2": 24, "Bytes1": 3811, "Bytes2": 28734, "Bandwidth1": 3811, "Bandwidth2": 28734 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "935227a1a3403545-08b0495f06426a29 (58532:4433)", "Ts": 1571077732254932, "State": "Up", "Full_rtt_responder": 78570, "Packets1": 11, "Packets2": 24, "Bytes1": 3811, "Bytes2": 28734, "Bandwidth1": 3811, "Bandwidth2": 28734 }
]
